examples/README.md

```diff
@@ -0,0 +1,17 @@
+## Example pull request workflow criteria
+
+1. Overview
+To automatically, periodically check the correctness of the example code, the github action is used to examine the code as the implementation of CI/CD(Continuous integration/continuous deployment) concept. This doc is to standardrize the example format. Nonstandard pull request(PR) will be blocked automatically. 
+2. Testing Design
+Several code formats are listed here.
+  A. The environment variable should be written in the file, not separately set in the terminal.
+  B. The dependent libraries should be listed in requirements.txt and add the command "pip install -r requirements.txt" in the test_ci.sh file. 
+  C. The config parameters should be small for fast testing
+  D. Add one test_ci.sh file to start the training. 
+3. File Structure Chart
```

Comment:
Take care of your Markdown format, proper line breaking is required for better readability.

Reply:
I see that this is copied from the design doc directly, please consider this `README.md` a doc for other contributors who are not part of the design process. They care mostly about the problem of how to test their example. Make sure you deliver this message clearly and make this doc easy to follow.

Reply:
My second and third comments are not handled.

Reply:
OK! I changed the README follow your instructions now!